rust/src/measurements/laplace/float/pseudocode/make_scalar_float_laplace_cks20.py

```diff
@@ -0,0 +1,22 @@
+# type: ignore
```

Comment:
Could add this to mypy excludes, I think. No change here, but filed an issue:
- #1329